StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1609SA1610CodeFixProvider.cs

```diff
@@ -0,0 +1,199 @@
+            else
+            {
+                DocumentationCommentTriviaSyntax newDocumentationComment = documentationComment.WithContent(
+                    documentationComment.Content.InsertRange(documentationComment.Content.Count - 1,
+                    XmlSyntaxFactory.List(
+                        leadingNewLine,
+                        valueElement)));
+
+                newRoot = root.ReplaceNode(documentationComment, newDocumentationComment);
+            }
+
+            return document.WithSyntaxRoot(newRoot);
+        }
+
+        private bool TryRemoveSummaryPrefix(ref SyntaxList<XmlNodeSyntax> summaryContent, string prefix)
+        {
+            XmlNodeSyntax firstContent = summaryContent.FirstOrDefault(IsContentElement);
+            XmlTextSyntax firstText = firstContent as XmlTextSyntax;
+            if (firstText != null)
+            {
+                string firstTextContent = string.Concat(firstText.DescendantTokens());
+                if (firstTextContent.TrimStart().StartsWith(prefix, StringComparison.Ordinal))
+                {
+                    // Find the token containing the prefix, such as "Gets or sets "
+                    SyntaxToken getsToken = default(SyntaxToken);
+                    foreach (SyntaxToken textToken in firstText.TextTokens)
+                    {
+                        if (textToken.IsMissing)
+                            continue;
+
+                        if (!textToken.Text.TrimStart().StartsWith(prefix, StringComparison.Ordinal))
+                            continue;
+
+                        getsToken = textToken;
+                        break;
+                    }
+
+                    if (!getsToken.IsMissing)
+                    {
+                        string text = getsToken.Text;
+                        string valueText = getsToken.ValueText;
+                        int index = text.IndexOf(prefix);
+                        if (index >= 0)
+                        {
+                            bool additionalCharacters = index + prefix.Length < text.Length;
+                            text = text.Substring(0, index)
+                                + (additionalCharacters ? char.ToUpperInvariant(text[index + prefix.Length]).ToString() : string.Empty)
+                                + text.Substring(index + (additionalCharacters ? (prefix.Length + 1) : prefix.Length));
+                        }
+
+                        index = valueText.IndexOf(prefix);
+                        if (index >= 0)
+                            valueText = valueText.Remove(index, prefix.Length);
+
+                        SyntaxToken replaced = SyntaxFactory.Token(getsToken.LeadingTrivia, getsToken.Kind(), text, valueText, getsToken.TrailingTrivia);
+                        summaryContent = summaryContent.Replace(firstText, firstText.ReplaceToken(getsToken, replaced));
+                        return true;
+                    }
+                }
+            }
+
```

Comment:
:point_up: The nested if statements above are not inline with the rest of the code, where an return as quickly as possible strategy is used.